### PR TITLE
Feathercoin Splashscreen and High resolution about Feathercoin banner

### PR DESCRIPTION
**Pull request by wrapperband**

### CODE REVIEW

File: src/qt/splashscreen.cpp [hunk 1]
```diff
@@ -42,15 +39,15 @@ SplashScreen::SplashScreen(Qt::WindowFlags f, const NetworkStyle *networkStyle)
 #endif
 
     // define text to place
-    QString titleText       = tr(PACKAGE_NAME);
+    QString titleText       = tr("Feathercoin Core");
     QString versionText     = QString("Version %1").arg(QString::fromStdString(FormatFullVersion()));
-    QString copyrightText   = QString::fromUtf8(CopyrightHolders(strprintf("\xc2\xA9 %u-%u ", 2013, COPYRIGHT_YEAR)).c_str());
```
Comment (FeatherCoin) [inline src/qt/splashscreen.cpp]: Doesn't the removal of CopyrightHolders remove the Bitcoin copyright info from the splash screen?

Comment (Bushstar) [inline src/qt/splashscreen.cpp]: Doesn't the removal of CopyrightHolders remove the Bitcoin copyright info from the splash screen?

Comment (wrapperband) [reply src/qt/splashscreen.cpp]: That wasn't noted as a problem with 0.9.6.2

Comment (Bushstar) [reply src/qt/splashscreen.cpp]: Sure, leave it as is then.

**Pull request merged**